_includes/sidebar.html

```diff
@@ -34,7 +34,7 @@ <h4 id="optional-steps">Optional steps</h4>
   </div>
 
   <div class="source">
-    <a href="https://spdx.org/licenses/{{ page.spdx-id }}.html">
+    <a href="https://spdx.org/licenses/{{ page.spdx-id }}{% if xgpl %}-only{% endif %}.html">
```

Comment:
I'd rather link to -or-later to subtly encourage that, as above (it's stated first in line 29).
```suggestion
    <a href="https://spdx.org/licenses/{{ page.spdx-id }}{% if xgpl %}-or-later{% endif %}.html">
```

Reply:
Long story, but: I initially resolved this via an identical change myself, and then a few moments ago I realized your comment included a code suggestion.

It's more robust & easier to review a suggestion made during review, since the reviewer should themselves know how accurate their fix was, and doesn't have to compare it to a more recent, different commit.

Fortunately GitHub seems to handle code suggestions well during underlying code changes -- the option to apply your suggestion became available again after I force pushed to remove my fix commit.